README.md

```diff
@@ -56,44 +61,93 @@ public function registerBlocks(): array
 -->
 
 
+## Block Definition
 
-### Block Definition
+Blocks are defined as `.block` files that consist of 2 to 3 parts:
 
-Blocks are defined as `.block` files that consist of 1 to 3 parts:
-- A YAML configuration section that defines the block's name, description, and other metadata
-as well as the block's properties and the form used to edit those properties
-- A PHP code section that allows for basic code to be executed when the block is rendered,
-similar to a partial
-- A Twig template section that defines the HTML markup template of the block
+- A YAML configuration section that defines the block's name, description, and other metadata as well as the block's properties and the form used to edit those properties.
+- A PHP code section that allows for basic code to be executed when the block is rendered, similar to a partial.
+- A Twig template section that defines the HTML markup template of the block.
 
-When there are two parts, they are the Settings & Markup sections. When there is just one part
-it is the Markup section.
+When there are two parts, they are the Settings (YAML) & Markup (Twig) sections.
 
-The following property values (name, description, etc) can be defined in the Settings section
-of the `.block` files:
+The following property values (name, description, etc) can be defined in the Settings (YAML) section of the `.block` files:
 
 ```yaml
 name: Example
 description: Example Block Description
 icon: icon-name
-ignoreContext: [] # List of contexts to never include the block in
-allowContext: [] # List of contexts to only include the block in
+context: [] # Defines the contexts in which this block can be used
```

Comment:
Could you provide some more examples for this, at least in this discussion thread? It sounds like your interpretation might be closer to "groups" or "tags" associated with a block that can then be used to filter the blocks; my original intention was to provide the context of the parent record (I.e. "page", "sidebar", "form", "post", etc). I suppose I might be able to use this system for a similar use, but it would be helpful to see how you're using it.

Reply:
Switched `context` terminology to `tags` as per discussions with the maintainers.